CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 0.10.6 ()
```

Comment:
FYI, these changelog entries actually get drafted as PRs are merged and then updated during the release process (https://github.com/hvac/hvac/blob/develop/CONTRIBUTING.md#creating--publishing-releases). Including them in this PR is no trouble either tho.